### PR TITLE
Fix issue 19887 creating new shipment: getting all trackers.

### PR DESCRIPTION
**Pull request by Nazar65**
### Description (*)
 After creating new shipment customer receive email with all my tracking number

### Fixed Issues (if relevant)
1. magento/magento2#19887: Issue title

### Manual testing scenarios (*)

1. after creating new shipment and exec $_shipment->getAllTracks()
2. in https://example.com/admin_xxxxx/admin/order_shipment/view/shipment_id/382
3. after tick Notify Customer by Email and click save
4. the customer receive email with all my tracking number

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @Nazar65. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-3755](https://jira.corp.magento.com/browse/ENGCOM-3755) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @Nazar65. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Comment by magento-engcom-team** [comment]
Hi @Nazar65. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

File: app/code/Magento/Sales/view/frontend/templates/email/shipment/track.phtml [hunk 1]
```diff
@@ -9,22 +9,23 @@
 ?>
 <?php $_shipment = $block->getShipment() ?>
 <?php $_order = $block->getOrder() ?>
-<?php if ($_shipment && $_order && $_shipment->getAllTracks()): ?>
-<br />
-<table class="shipment-track">
-    <thead>
+<?php $trackCollection = $_order->getTracksCollection($_shipment->getId()) ?>
```
Comment (apedicdev) [inline app/code/Magento/Sales/view/frontend/templates/email/shipment/track.phtml:12]: `getTracksCollection ()` doesn't accept argument, does it?

**Pull request merged**